tests/Integration/Metadata/GetCurrentUrlTest.php

```diff
@@ -9,7 +9,7 @@
 
 namespace Parsely\Tests\Integration\Metadata;
 
-use Parsely\Metadata\Front_Page_Builder;
+use Parsely\Metadata\{ Category_Builder, Front_Page_Builder };
```

Comment:
Across the project, we don't use bracket syntax in `use`s, but I'm not against this. If we still need to import `Front_Page_Builder`, let's make sure this syntax is PHP 7.4 compatible though.

Alternatively, it can be broken into 2 `use`s.